### ISSUE TITLE
item.usage() returns None when using the default output (as_df=True). Version 2.0.0

### ISSUE THREAD

**Issue by BoudewijnBoogaard** [Esri/arcgis-python-api#1205]
**Describe the bug**
The arcgis.gis.Item.usage method returns None when using the default output (as_df=True). 

The issue is caused by an error that is thrown in file site-packages\arcgis\gis\__init__.py on line 12468. Because the error is thrown it goes into the except block and returns None.

The original line is:
df.Date = df.astype(float) / 1000

Changing it to this seems to fix the issue:
df.Date = df.Date.astype(float) / 1000

When using this code in ArcGIS Online Notebooks it does return a proper result. So it seems to be fixed there.

**To Reproduce**
Login to ArcGIS
Get an item
Get the usage info for that item using item.usage()
Set a breakpoint on line 12468 in the __init__.py and run the line df.Date = df.astype(float) / 1000 in the console

```python
from arcgis import GIS

gis = GIS(profile="")
item = gis.content.get('')
usage = item.usage()
print(usage, type(usage))

```
error:
```python
Traceback (most recent call last):
  File "C:\Users\bbo.EAGLE\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\lib\site-packages\IPython\core\interactiveshell.py", line 3437, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-1-926fd7343789>", line 1, in <module>
    df.Date = df.astype(float) / 1000
  File "C:\Users\bbo.EAGLE\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\lib\site-packages\pandas\core\generic.py", line 5516, in __setattr__
    self[name] = value
  File "C:\Users\bbo.EAGLE\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\lib\site-packages\pandas\core\frame.py", line 3602, in __setitem__
    self._set_item_frame_value(key, value)
  File "C:\Users\bbo.EAGLE\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\lib\site-packages\pandas\core\frame.py", line 3729, in _set_item_frame_value
    raise ValueError("Columns must be same length as key")
ValueError: Columns must be same length as key
```

**Expected behavior**
It should return a pandas dataframe with the usage information

**Platform (please complete the following information):**
 - OS: Windows

 - Python API Version 2.0.0.0 (also noticed it in 1.9.1)

**Comment by nanaeaubry** [comment]
Thanks for finding this bug, we will address it for the next release

**Comment by nanaeaubry** [comment]
Fixed in 2.0.1
